sanitizers/sanitizers.bzl

```diff
@@ -24,6 +24,7 @@ _sanitizer_class_names = [
     "RegexInjection",
     "RegexRoadblocks",
     "SqlInjection",
+    "XpathInjection",
```

Comment:
You can either format via `./format.sh` locally or apply the diff printed by the [failing CI job](https://github.com/CodeIntelligenceTesting/jazzer/runs/7841422789?check_suite_focus=true#step:5:15).

Reply:
Yeah sorry for that, somehow format.sh changes something in pretty much every file of the project so I assumed it does not work correctly for me. Applied the diff changes hope that it's correct.

Reply:
Oops, that's not nice. Could you give me an example? Then I will try to fix it.

Reply:
I'm sure my configuration is at fault. It for example removed every comma of the last element e.g. 

 "RegexRoadblocks",
 "SqlInjection",
-"XpathInjection",
+"XpathInjection"

I now installed everything without homebrew and it seems not work.